Dockerfile

```diff
@@ -1,22 +1,40 @@
 # This Dockerfile creates an environment suitable for downstream usage of AllenNLP.
-# It creates an environment that includes a pip installation of allennlp.
+# It creates an environment that includes a pip installation of allennlp-server.
 
-FROM python:3.6.8-stretch
+FROM python:3.6-stretch
```

Comment:
I think ideally this image should start with allennlp's. But I didn't find master pushed on Docker Hub. Thoughts?